bench/perf_run_py/perf_stat.py

```diff
@@ -153,12 +153,11 @@ def get_value_list(self, brief: bool):
 class PerfStatLatencyWorkgen(PerfStat):
     def get_value(self, nth_max: int):
         """Return the nth maximum number from all the gathered values"""
-        return sorted(self.values)[-nth_max]
+        return sorted(self.values)[-nth_max] if nth_max < len(self.values) else 0
 
     def get_value_list(self, brief: bool):
         as_list = []
-        num_max = min(len(self.values), 5)
-        for i in range(1, num_max + 1):
+        for i in range(1, 6):
```

Comment:
In the case that there is 0 values, then shouldn't the list be zero? And does the old code achieve the same with the new code?

Reply:
If we have 0 values:

- Before these changes, we would not print the top 5 latencies. 
- After these changes, we will print the top 5 latencies, and they will be 0.

Refer to @ajmorton's comment in the ticket to find out why this is necessary.

Reply:
"My preference is to report these stats, as it's a useful data point in our charts to see when no latency warnings are reported. We should also review other aggregate stats to see if they need to handle the same case"

Reply:
> We should also review other aggregate stats to see if they need to handle the same case
Just to double check @Siddhartha8899 did we find any other aggregate stats that may need the same treatment? My assumption is no as our other stats tend to be much larger numbers compared to numLatencyWarnings

Reply:
Yes your assumption is correct @ajmorton

Reply:
Thanks for explanation 🍔